booking-up-for-beauty/Pratster95/booking_up_for_beauty.go

```diff
@@ -0,0 +1,46 @@
+package booking
+
+import "time"
+import "fmt"
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	myDate, _ := time.Parse("1/2/2006 15:04:05", date)
+	return myDate
+	panic("Please implement the Schedule function")
+}
+
+// HasPassed returns whether a date has passed
+func HasPassed(date string) bool {
+	today := time.Now()
+	//myDateString := "July 25, 2019 13:45:00"
+	myDate, _ := time.Parse("January 2, 2006 15:04:05", date)
+	return today.After(myDate)
+	panic("Please implement the HasPassed function")
```

Comment:
Same, careful with code after `return`. The tests do not include failing errors, which you can try to test yourself to see how this `panic` behaves and if it is raised anytime.